### ISSUE TITLE
Notebook 06-flow-controls

### ISSUE THREAD

**Issue by KevinMurphy806** [dpploy/chen-3170#18]
Not an issue within the notebook, but possibly increase the amount of assert statements that should be included in the labs (at least the earlier ones).  Those can be useful in teaching why certain code doesn't work besides the program just giving an error.  Also, I would teach while loops along with the rest of the information in this notebook, especially since we never got around to really using them and they can be very useful for looping code.

**Comment by dealmeidavf** [comment]
Good point. The `assert` statement is useful and I should use them early in the course. I will expand on the `while` loop control.
